content/dart/concepts/type-conversion/type-conversion.md

```diff
@@ -0,0 +1,496 @@
+  var c = (2351.44464661).toStringAsPrecision(2);
+  var d = 254645466111.toStringAsPrecision(3);
+  var e = 1224.toStringAsPrecision(20);
+  var f = 5.25.toStringAsPrecision(12);
+
+  print(a);
+  print(b);
+  print(c);
+  print(d);
+  print(e);
+  print(f);
+
+}
+```
+
+The `Shell` output:
+
+```Shell
+1.00
+3e+5
+2.4e+3
+2.55e+11
+1224.0000000000000000
+5.25000000000
+```
+
+---
+
+## toDouble() Method
+
+The `toDouble()` method converts a value to a double.
+
+## toDouble Method Example
+
+```dart
+void main() {
+  int intValue = 42;
+  double doubleValue = intValue.toDouble();
+
+  var checkCondition = (doubleValue == 42.0) ? true : false;
+
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
+## toInt() Method
+
+The `toInt()` method converts a value to an integer.
+
+## toInt Method Example
+
+```dart
+void main() {
+  double doubleValue = 42.75;
+  int intValue = doubleValue.toInt();
+
+  var checkCondition = (intValue == 42) ? true : false;
+
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
+## parse() Method
+
+The `parse()` method converts a String to a numeric type. Parses a string containing a number literal into a number.
+
+### parse() Method Variations
+
+### `num.parse()`
+
+  The method first tries to read the input as `integer` (similar to `int.parse` without a `radix`). If that fails, it tries to parse the input as a `double` (similar to `double.parse`). If that fails, too, it throws a `FormatException`.
+
+### `int.parse()`
+
+  Converts a String to an `integer` type value. If that fails, too, it throws a `FormatException`.
+
+### `double.parse()`
+
+  Converts a String to an `double` type value. If that fails, too, it throws a `FormatException`.
+
+## parse Method Example
+
+```dart
+void main() {
+  String stringValue = "42";
+  int intValue = int.parse(stringValue);
+
+  var checkCondition = (intValue.runtimeType == int) ? true : false;
+  print(checkCondition);
+}
+```
+
+The `Shell` output:
+
+```Shell
+true
+```
+
+---
+
+## tryParse() Method
+
+The `tryParse()` method converts a String to a numeric type, similar to `parse()`, but returns `null` if the conversion fails.
+
+### tryParse() Method Variations
+
+### `num.tryParse()`
+
+  The method first tries to read the input as `integer` (similar to `int.tryParse` without a `radix`). If that fails, it tries to parse the input as a `double` (similar to `double.tryParse`). If that fails, too, the function returns `null` instead of throwing `FormatException`.
+
+### `int.tryParse()`
+
+  Converts a String to an `integer` type value. For invalid input, the function returns `null`.
+
+### `double.tryParse()`
+
+  Converts a String to an `double` type value. For invalid input, the function returns `null`.
+
+## tryParse Method Example
+
+```dart
+void main() {
+  print(int.tryParse('42'));
+  print(int.tryParse('42f'));
+}
+```
+
+The `Shell` output:
+
+```Shell
+42
+null
+```
+
+---
+
```

Comment:
We don't need any specific function explanation or example in this case. Please remove the above-mentioned lines.